### ISSUE TITLE
Please modify the script to work without USB

### ISSUE THREAD

**Issue by vitaly-zdanevich** [BWGosselin/Ventoy_GRUB_Entry#1]

**Comment by BWGosselin** [comment]
What do you mean by that? Like to work on a hdd/sdd? It was intended to work with a usb

**Comment by vitaly-zdanevich** [comment]
> It was intended to work with a usb

Yep, but maybe this is not very difficult to add it to GRUB somehow :(

**Comment by BWGosselin** [comment]
So basically you want to download ventoy onto a hdd/sdd and for a entry to appear in the grub screen?

**Comment by vitaly-zdanevich** [comment]
Yes, to use Ventoy without USB and without a new partition.

**Comment by BWGosselin** [comment]
Okay that's a great idea! Give me a couple days and I'll have it updated. Busy with life atm.

**Comment by jesudiaa** [comment]
I have installed Ventoy on my internal hard drive and on another partition I have installed Debian 12 but Debian overwrites the MBR and I can only boot Debian. I want to include the Ventoy installed on the hard drive in Grub2. Your script doesn't work for me. What solution could there be to have Ventoy boot from my Debian 12?

**Comment by jesudiaa** [comment]
I have installed Ventoy on my internal hard drive and on another partition I have installed Debian 12 but Debian overwrites the MBR and I can only boot Debian. I want to include the Ventoy installed on the hard drive in Grub2. Your script doesn't work for me. What solution could there be to have Ventoy boot from my Debian 12?
